ds701_book/21-Networks-I.qmd

```diff
@@ -253,422 +387,409 @@ nx.draw_networkx(DG,
+:::
+::::
 
 More surprising was the fact that the __average path length was 6.2 people!__
 
-This statistic became famous when John Guare wrote a play called _Six Degrees of Separation._
+---
+
+This statistic became famous when John Guare wrote a play called _Six Degrees of Separation_.
 
-Given what we know about clustering in social networks, this is quite surprising.   How can we explain it?
+Given what we know about clustering in social networks, this is quite surprising. How can we explain it?
 
 The first clue comes from another classic social science paper, called _The Stength of Weak Ties,_ by Mark Granovetter.
 
-This is sometimes referred to as the most famous paper in sociology (60,000 citations!).   
+This is sometimes referred to as the most famous paper in sociology (with over 60,000 citations).   
 
-Granovetter interviewed people about how the found their jobs.  He found that most people did not get a job through someone that was a close friend, but rather through a __distant__ acquaintance.
+Granovetter interviewed people about how the found their jobs. He found that most people did not get a job through someone that was a close friend, but rather through a __distant__ acquaintance.
 
 This suggests that an important way that information travels in a social network is via the rare connections that exist __outside__ of the local clustering of friendships.
 
+---
+
 This was all put on an experimental basis in __another__ classic paper, by the social scientist Duncan Watts and the mathematician Steve Strogatz.
-<center>
-    
-<img src="figs/duncan_watts.jpeg" alt="Figure" width="200px" align="left">
-<img src="figs/steven-strogatz.png" alt="Figure" width="200px" align="left">
-    
-</center>
 
 In their paper _Collective Dynamics of Small-World Networks_, Watts and Strogatz perfomed an elegant experiment.
 
-They asked: if we take a highly clustered network, and slightly perturb it (randomize it), what happens?
+They asked, what if we take a highly clustered network, and slightly perturb it?
+
+---
 
 Specifically, they started with a network in which each node is connected to a fixed number of neighbors, and connections are made in a __highly clustered__ way.
 
-<center>
-    
-<img src="figs/L21-watts-strogatz.png" alt="Figure" width="60%">
-    
-</center>
+![](figs/L21-watts-strogatz.png){fig-align="center" width="60%"}
 
 Then, with probability $p$, __rewire__ each edge: change it to connect to a __random__ destination.
 
+As $p$ varies, what happens to
 
-So, as $p$ varies, what happens to:
-* Average path length $L(p)$ and
-* Clustering coefficient $C(p)$?
+* the average path length $L(p)$ and
+* the clustering coefficient $C(p)$?
 
-And here is the famous figure from that paper:
+---
+
+Here is the famous figure from that paper. Observe the log scale on the $p$ axis.
    
-<center>
-    
-<img src="figs/watts-strogatz-2.png" alt="Figure" width="700px">
-    
-</center> 
+![](figs/watts-strogatz-2.png){width="700px"}
```

Comment:
Too bad the vertical axis is not labeled clearly. Is it L(p)/L(0)?

Reply:
I added a line that the y-axis shows L(p)/L(0) and C(P)/C(0).